wazo_dird/plugins/phonebook_backend/schemas.py

```diff
@@ -26,3 +32,19 @@ class ListSchema(_ListSchema):
 source_list_schema = SourceSchema(many=True)
 source_schema = SourceSchema()
 list_schema = ListSchema()
+
+
+class CountParams(TypedDict):
+    search: str | None
+
+
+class ContactListSchema(_ListSchema):
+    searchable_columns = ['uuid', 'firstname', 'lastname']
+    sort_columns = ['firstname', 'lastname']
```

Comment:
* `GET /phonebooks/uuid/contacts` allows `order` = `firstname` or `lastname` or `name`. Any idea why `name`? Can we homogenize the two API?